extras/kms/examples/oidc.go

```diff
@@ -12,8 +12,8 @@ type OIDC struct {
 	CtClientSecret []byte `json:"-" gorm:"column:client_secret" wrapping:"ct,client_secret"`
 	// ClientSecret is the oidc client secret (plaintext)
 	ClientSecret string `json:"client_secret,omitempty" wrapping:"pt,client_secret"`
-	// KeyId is the wrapper key id used to encrypt/decrypt the client secret
-	KeyId string `json:"key_id,omitempty" gorm:"not_null"`
+	// KeyVersionId is the wrapper key version id used to encrypt/decrypt the client secret
+	KeyVersionId string `json:"key_id,omitempty" gorm:"not_null"`
```

Comment:
Leaning towards changing the json tag here as well to make it explicit that it is a key version that is referenced.